
from PyQt5 import QtCore, QtGui, QtWidgets


class Ui_adminWindow(object):
    def setupUi(self, adminWindow):
        adminWindow.setObjectName("adminWindow")
        adminWindow.resize(1200, 800)
        adminWindow.setMinimumSize(QtCore.QSize(1200, 800))
        adminWindow.setMaximumSize(QtCore.QSize(1200, 800))
        self.tableWidget = QtWidgets.QTableWidget(adminWindow)
        self.tableWidget.setGeometry(QtCore.QRect(300, 110, 891, 541))
        self.tableWidget.setObjectName("tableWidget")
        self.tableWidget.setColumnCount(0)
        self.tableWidget.setRowCount(0)
        self.verticalLayoutWidget = QtWidgets.QWidget(adminWindow)
        self.verticalLayoutWidget.setGeometry(QtCore.QRect(0, 110, 291, 541))
        self.verticalLayoutWidget.setObjectName("verticalLayoutWidget")
        self.verticalLayout = QtWidgets.QVBoxLayout(self.verticalLayoutWidget)
        self.verticalLayout.setContentsMargins(0, 0, 0, 0)
        self.verticalLayout.setObjectName("verticalLayout")
        self.pushButton_3 = QtWidgets.QPushButton(self.verticalLayoutWidget)
        font = QtGui.QFont()
        font.setFamily("Arial")
        font.setPointSize(18)
        self.pushButton_3.setFont(font)
        self.pushButton_3.setObjectName("pushButton_3")
        self.verticalLayout.addWidget(self.pushButton_3)
        self.pushButton_5 = QtWidgets.QPushButton(self.verticalLayoutWidget)
        font = QtGui.QFont()
        font.setFamily("Arial")
        font.setPointSize(18)
        self.pushButton_5.setFont(font)
        self.pushButton_5.setObjectName("pushButton_5")
        self.verticalLayout.addWidget(self.pushButton_5)
        self.pushButton = QtWidgets.QPushButton(self.verticalLayoutWidget)
        font = QtGui.QFont()
        font.setFamily("Arial")
        font.setPointSize(18)
        self.pushButton.setFont(font)
        self.pushButton.setObjectName("pushButton")
        self.verticalLayout.addWidget(self.pushButton)
        self.pushButton_2 = QtWidgets.QPushButton(self.verticalLayoutWidget)
        font = QtGui.QFont()
        font.setFamily("Arial")
        font.setPointSize(18)
        self.pushButton_2.setFont(font)
        self.pushButton_2.setObjectName("pushButton_2")
        self.verticalLayout.addWidget(self.pushButton_2)
        self.pushButton_6 = QtWidgets.QPushButton(self.verticalLayoutWidget)
        font = QtGui.QFont()
        font.setFamily("Arial")
        font.setPointSize(18)
        self.pushButton_6.setFont(font)
        self.pushButton_6.setObjectName("pushButton_6")
        self.verticalLayout.addWidget(self.pushButton_6)
        self.pushButton_7 = QtWidgets.QPushButton(self.verticalLayoutWidget)
        font = QtGui.QFont()
        font.setFamily("Arial")
        font.setPointSize(18)
        self.pushButton_7.setFont(font)
        self.pushButton_7.setObjectName("pushButton_7")
        self.verticalLayout.addWidget(self.pushButton_7)
        self.pushButton_4 = QtWidgets.QPushButton(adminWindow)
        self.pushButton_4.setGeometry(QtCore.QRect(1108, 0, 91, 51))
        font = QtGui.QFont()
        font.setFamily("Arial")
        font.setPointSize(18)
        self.pushButton_4.setFont(font)
        self.pushButton_4.setObjectName("pushButton_4")
        self.label = QtWidgets.QLabel(adminWindow)
        self.label.setGeometry(QtCore.QRect(330, 70, 91, 31))
        font = QtGui.QFont()
        font.setFamily("Arial")
        font.setPointSize(20)
        self.label.setFont(font)
        self.label.setObjectName("label")
        self.textEdit = QtWidgets.QTextEdit(adminWindow)
        self.textEdit.setGeometry(QtCore.QRect(430, 70, 641, 31))
        font = QtGui.QFont()
        font.setFamily("Arial")
        self.textEdit.setFont(font)
        self.textEdit.setObjectName("textEdit")
        self.addButton = QtWidgets.QPushButton(adminWindow)
        self.addButton.setGeometry(QtCore.QRect(390, 680, 91, 35))
        font = QtGui.QFont()
        font.setFamily("Arial")
        font.setPointSize(18)
        self.addButton.setFont(font)
        self.addButton.setObjectName("addButton")
        self.delButton = QtWidgets.QPushButton(adminWindow)
        self.delButton.setGeometry(QtCore.QRect(630, 680, 91, 35))
        font = QtGui.QFont()
        font.setFamily("Arial")
        font.setPointSize(18)
        self.delButton.setFont(font)
        self.delButton.setObjectName("delButton")
        self.pushButton_10 = QtWidgets.QPushButton(adminWindow)
        self.pushButton_10.setGeometry(QtCore.QRect(840, 680, 91, 35))
        font = QtGui.QFont()
        font.setFamily("Arial")
        font.setPointSize(18)
        self.pushButton_10.setFont(font)
        self.pushButton_10.setObjectName("pushButton_10")

        self.retranslateUi(adminWindow)
        QtCore.QMetaObject.connectSlotsByName(adminWindow)

    def retranslateUi(self, adminWindow):
        _translate = QtCore.QCoreApplication.translate
        adminWindow.setWindowTitle(_translate("adminWindow", "Form"))
        self.pushButton_3.setText(_translate("adminWindow", "Customer Mangment"))
        self.pushButton_5.setText(_translate("adminWindow", "Transaction management"))
        self.pushButton.setText(_translate("adminWindow", "Bill management"))
        self.pushButton_2.setText(_translate("adminWindow", "Payment management"))
        self.pushButton_6.setText(_translate("adminWindow", "Transaction history"))
        self.pushButton_7.setText(_translate("adminWindow", "Report generation"))
        self.pushButton_4.setText(_translate("adminWindow", "exit"))
        self.label.setText(_translate("adminWindow", "Search"))
        self.addButton.setText(_translate("adminWindow", "Add"))
        self.delButton.setText(_translate("adminWindow", "Delete"))
        self.pushButton_10.setText(_translate("adminWindow", "Update"))
